skel/Makefile

```diff
@@ -3,7 +3,7 @@ APPS     :=	sonic
 all:	$(APPS)
 
 $(APPS):
-	go build -o bin/$@ app.go
+	go build --ldflags "-X main.Version=$$(cat .tags)" -o bin/$@ app.go
```

Comment:
Can you explain please?

Reply:
The `--ldflags` will insert dynamic information into the binary at build time with .tags (created in tagging of ci process) value.